.github/actions/draft-release/entrypoint.sh

```diff
@@ -24,8 +24,10 @@ touch temp-changes.txt
     echo "## $RELEASE_NAME"
     echo ""
     git log "$LAST_TAG"..HEAD --no-merges --oneline --pretty="format:- %s" --perl-regexp --author='^((?!dependabot).*)$'
-    echo $'\n'
+    echo $'\n- Dependency updates\n'
```

Comment:
I was wondering if there were any extreme edge cases where we could have one of these changelogs generated for a delta that didn't actually contain any dependency updates  (e.g. ad-hoc release shortly before scheduled release, preceding another dependabot round). I think there may be some remote possibility, but it seems sufficiently unlikely to happen in reality, and even if it did, there's no substantive impact anyway

Reply:
If so, it can be removed before merging. Same as we'd remove a commit message about updated documentation or whatever..More often than not, it is going to be useful to not add it by hand

Reply:
Completely agreed!